src/xpath.c

```diff
@@ -9260,8 +9252,11 @@ eval_equality_expr(const struct lyxp_expr *exp, uint32_t *tok_idx, uint32_t repe
             lyxp_set_scnode_merge(set, &set2);
             set_scnode_clear_ctx(set, LYXP_SET_SCNODE_ATOM_VAL);
         } else {
-            rc = moveto_op_comp(set, &set2, &exp->expr[exp->tok_pos[this_op]]);
+            ly_bool result;
+
+	    rc = moveto_op_comp(set, &set2, &exp->expr[exp->tok_pos[this_op]], &result);
```

Comment:
Had to run `uncrustify` myself, you have a TAB here.

Reply:
Sorry, couldn't install `uncrustify` on my system due to the minimum required version being to high.